ui-spacetimechart/src/types.ts

```diff
@@ -0,0 +1,63 @@
+export type WorkSchedule = {
+  type: 'TRACK' | 'CATENARY';
+  timeStart: Date;
+  timeEnd: Date;
+  spaceRanges: [number, number][];
+};
+
+export type OperationalPointType = ArrayElement<PathProperties['operational_points'] | null>;
```

Comment:
Hm, seems like all of the types below are re-defined? They're already in ui-manchette, but they're not used in ui-spacetimechart?

Reply:
I tidied up all this, only using the existing spacetime types for this story